embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/gating/SpanSanitizer.kt

```diff
@@ -38,17 +39,11 @@ internal class SpanSanitizer(
     }
 
     private fun sanitizeSpans(span: EmbraceSpanData): Boolean {
-        if (span.name == "emb-${SchemaKeys.VIEW_BREADCRUMB}" && !shouldAddViewBreadcrumbs()) {
-            return false
-        }
-        return true
+        return !(span.hasEmbraceAttribute(EmbType.Ux.View) && !shouldAddViewBreadcrumbs())
```

Comment:
nit: would marginally prefer the previous pattern. While this is a reasonable thing to change for just two checks, the gating will make dozens of conditional checks

Reply:
Got it. So you refer to use the early return pattern? I can restore this. Agreed that if there are more than 3 checks, this is a bad idiom to follow.

But given there's another change that builds on this. is it OK if I restored this pattern post-merge?